### ISSUE TITLE
Reproduction Issue: Obtained Results Lower Than Reported

### ISSUE THREAD

**Issue by veronikayurchuk** [tulerfeng/Video-R1#65]
Thanks for releasing this great work and open-sourcing the code! I attempted to reproduce the results shown in the repo but obtained noticeably lower scores on several benchmarks.

Setup
- Hardware: 4 × NVIDIA A100 80 GB
- Software: unmodified commit [639903f](https://github.com/tulerfeng/Video-R1/commit/639903fa28cad5128402dc957d37943f59914eda) with the same unchanged default hyper-parameters and followed all instructions
torch==2.5.1
cuda v12.4
vllm==0.7.2
trl==0.16.0
- I skipped the SFT process and used the model provided in the repository. For RL, I ran training up to 1500 steps, resuming once at step 100 and again at step 800 (with no other changes).

|            | MMVU          | MvBench       | TempCompass   | VideoMME      | VideoMMMU     | VsiBench      |
|------------|---------------|---------------|---------------|---------------|---------------|---------------|
| **Repo**   | 64.2          | 62.7          | 72.6          | 57.4          | 49.8          | 34.6          |
| **checkpoint-1200**   | 64.16   | —             | 69.73  | 53.48   | 48.44   | 30.28 |

WandB screenshots
(My training/reward & train/accuracy-reward function does not appear to increase in the same way as shown in the paper or read.me .)
<img width="1628" height="856" alt="Image" src="https://github.com/user-attachments/assets/05d1dce0-b4a6-4462-9086-bcc75e5bfc3d" />
<img width="1626" height="853" alt="Image" src="https://github.com/user-attachments/assets/416eecf7-0bfa-424c-a995-c4ba45136c68" />

Questions:
- Are there any non-default flags, environment variables, or dependencies that could affect training accuracy?
- Did you apply any additional filtering or ensembling during evaluation?
- Do you have any ideas on what might cause the lower scores or the flatter reward curve?

Any insights or suggestions would be greatly appreciated!

**Comment by tulerfeng** [comment]
Hi, thanks for your question.

From the curves, things appear to be progressing reasonably well.

 > accuracy-reward function does not appear to increase in the same way as shown in the paper or read.me

You may set the range of y-axis to $0.5\sim0.8$ so that you can see the upward trend more clearly, as illustrated in the paper.

> Are there any non-default flags, environment variables, or dependencies that could affect training accuracy?

I didn't test on other versions of libraries. But based on my experience, the accuracy couldn't be affected so much in terms of these variables. However, the max_frame and resolution should be set properly.

> Did you apply any additional filtering or ensembling during evaluation?

No. Directly running the provided evaluation scripts is OK.

> Do you have any ideas on what might cause the lower scores or the flatter reward curve?

1. Can you obtain correct accuracy running the provided Video-R1-7B model (https://huggingface.co/Video-R1/Video-R1-7B) in your environment?
2. You may try to evaluate on checkpoints at other steps (e.g., 800, 1000) to see the performance. Sometimes there are maybe gaps between models obtained by different steps. 
3.  For VSI-Bench, the performance is calculated by (mean_acc+mean_mra) / 2. 

Feel free to contact me for any feedbacks.
